lib/utils.js

```diff
@@ -530,5 +531,9 @@ module.exports = {
   getPatientContact: (db, patientShortcodeId, callback) => {
     getPatient(db, patientShortcodeId, true, callback);
   },
+  isValidBooleanExpression: expr =>
+    typeof expr === 'string' && expr.trim() !== '',
+  evalExpression: (context, expr) =>
+    vm.runInNewContext(expr, context),
```

Comment:
Switching the order of params is confusing to me - `expr`, `context` is the right order IMO.

Reply:
I was keeping the order that the things I was refactoring used. I'll switch it around